core/src/main/java/io/confluent/rest/metrics/MetricsResourceMethodApplicationListener.java

```diff
@@ -383,15 +405,15 @@ public void finished(long requestSize, long responseSize, long latencyMs) {
 
       if (enableLatencySloSla) {
         if (latencyMs < latencySloMs) {
-          responseLatencySloSlaSensor.get(LATENCY_SLO_SLA_SENSOR_NAMES[0]).record();
+          responseLatencySloSlaSensors.get("response-below-latency-slo").record();
```

Comment:
I think these names ought to be in final static strings. Apart from that, I think this PR is looking good.

Reply:
Make sense.